Makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY: build clean test doc
+.PHONY: build clean test doc install
```

Comment:
I would be in favor of removing the `Makefile` and using `dune` directly.

Reply:
I'd prefer to keep the makefile -- I use the convenience targets all the time :-)